src/views/order_list_view.js

```diff
@@ -0,0 +1,153 @@
+  events:{
+    'click button.btn-buy, button.btn-sell': 'addOrder',
+  },
+
+  // function to draw the order form dropdown selection options
+  renderOrderForm() {
+    // add symbols to dropdown menu in the form
+    const quoteData = this.quotes.map( x => x.get('symbol'));
+    quoteData.forEach((symbol) => {
+      const option = `<option value="${symbol}">${symbol}</option>`;
+      this.$('form select').append(option);
+    });
+  },
+
+  // function that gets called when buy or sell button in view get clicked on, will validate and add an order or make a call to show user error messages
+  addOrder(event){
+    console.log('inside addOrder method');
+    event.preventDefault();
+    // get form data
+    const formData = this.getFormData();
+    formData['buy'] = event.target.classList[0] === ("btn-buy")  ? true : false;
+
+    // new instance of OrderView using form formData
+    const newOrder = new Order(formData);
+
+    if (newOrder.isValid()){
+      // add order and clear form formData
+      console.log('new order is valid!');
+      this.model.add(newOrder);
+      const action = newOrder.get('buy') ? 'buy' : 'sell';
+      this.updateStatusMessage(`Successfully added a ${action} open order for ${newOrder.get('symbol')}`);
+      this.clearFormData();
+      // console.log(newOrder);
+    } else {
+      console.log('new order is invalid!');
+      // get rid of task and give user feedback on error handling
+      this.updateStatusMessage(newOrder.validationError)
+      newOrder.destroy();
+    }
+  },
+
+  // helper function to get symbol and target price from the form
+  getFormData() {
+    console.log('in getFormData method');
+    let orderData = {};
+
+    orderData['symbol'] = this.$(`#order-form select[name="symbol"]`).val();
+    orderData['targetPrice'] = Number(this.$('#order-form input[name="price-target"]').val());
+
+    // get the current price of the quote
+    orderData['quotePrice'] = this.quotes.findWhere({'symbol': orderData['symbol']}).get('price');
+
+    return orderData;
+  },
+
+  // function to clear form
+  clearFormData() {
+    console.log('in clearFormData');
+    // reset to the first quotes symbol and clear price value
+    this.$(`#order-form select[name="symbol"]`).val(this.quotes.models[0].get('symbol'));
+    this.$('#order-form input[name="price-target"]').val('');
+  },
+
+  // method to show validation error messages or success messages to the user based on their form input
+  updateStatusMessage(message) {
+    // jQuery select spot where we want to add messages
+    const $statusMessages = this.$('.form-errors');
+    // clear the messages
+    $statusMessages.empty();
+    // append the message to the DOM
+    $statusMessages.append(`<p>${message}</p>`);
+  },
+
+  // this gets called when prices are updated. It will check if the price change reaches a point where it should trigger buying or selling a quote, as well as removing the order
+
+  orderToQuote(quote){
+    let orders;
+
```

Comment:
Since this function only modifies the models / collections and doesn't do any view-specific things, it would be a good candidate for an instance method on the `OrderList` collection.